utils/message_scheduler.py

```diff
@@ -46,7 +46,7 @@ async def send_daily_question_and_update_stats_schedule() -> None:
     await send_daily_question_and_update_stats()
 
 
-async def send_daily_question_and_update_stats(update_stats: bool = True, force_daily_reset: bool = False, force_weekly_reset: bool = False) -> None:
+async def send_daily_question_and_update_stats(update_stats_bool: bool = True, force_daily_reset: bool = False, force_weekly_reset: bool = False) -> None:
```

Comment:
Don't think I agree with the name change.

Reply:
I believe I called this due to the function update_stats being called instead, causing issues, not sure if this is still here but feel free to revert if not

Reply:
This is why Unit / Automated Tests would've been great. But I'm lazy :(